packages/create-cloudflare/src/frameworks/next/index.ts

```diff
@@ -94,6 +95,8 @@ const configure = async (ctx: PagesGeneratorContext) => {
 		startText: "Adding the Cloudflare Pages adapter",
 		doneText: `${brandColor(`installed`)} ${dim(packages.join(", "))}`,
 	});
+
+	writeFile(path, nextConfig);
```

Comment:
Spits back error:

`Error: ENOENT: no such file or directory, open 'next-bindings-c3-test/src/app'`

Though this is correct directory. Think I don't quite follow sequence of operations within create cloudflare.

Reply:
This does the trick:
```ts
	writeFile(`${ctx.project.path}/next.config.js`, nextConfig);
```

in your code `path` represents a directory so I wouldn't expect it to work (but the error message is a bit baffling 😕)

Also the next.config.js file should be at the root of the project and not within the app directory 🙂